.github/workflows/build-multi-stage.yaml

```diff
@@ -15,7 +15,13 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: multi-arch-build
+        # yamllint disable-line rule:line-length
+        if: ${{ ! contains(github.event.pull_request.labels.*.name, 'ci/skip/multi-arch-build') }}
```

Comment:
@nixpanic instead of running a single arch how about doing `exit 0` or `return 0` which makes it even faster and really helps in the doc change or yaml change?

Reply:
I considered something like that too. It seems this just take ~3 minutes now, so from my point of view that is acceptable. Do you want me to change this?

Reply:
should be fine :+1: